shared/src/test/java/uk/gov/di/authentication/shared/serialization/MfaDetailDeserializerTest.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.di.authentication.shared.serialization;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.junit.jupiter.api.Test;
+import uk.gov.di.authentication.shared.entity.AuthAppMfaDetail;
+import uk.gov.di.authentication.shared.entity.MFAMethodType;
+import uk.gov.di.authentication.shared.entity.MfaDetail;
+import uk.gov.di.authentication.shared.entity.SmsMfaDetail;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Nitpick - [avoid wildcard imports](https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages/java.html#imports) (that guidance has the way of making sure inteliij doesn't do this automatically for you - I think a recent intellij update reset these settings for me so might be the same for you)